packages/rulesets/generated/spectral/az-dataplane.js

```diff
@@ -754,20 +790,20 @@ const ruleset$1 = {
                 function: falsy,
             },
         },
-        "az-api-version-enum": {
+        'ApiVersionEnum': {
             description: "The api-version parameter should not be an enum.",
             severity: "warn",
             formats: [oas2, oas3],
             given: [
-                "$.paths[*].parameters.[?(@.name == 'api-version')]",
-                "$.paths.*[get,put,post,patch,delete,options,head].parameters.[?(@.name == 'api-version')]",
+                "$.paths[*].parameters.[?(@.name == 'ApiVersion')]",
+                "$.paths.*[get,put,post,patch,delete,options,head].parameters.[?(@.name == 'ApiVersion')]",
```

Comment:
I don't think you want to change this -- the parameter name should be 'api-version'.
```suggestion
                "$.paths[*].parameters.[?(@.name == 'api-version')]",
                "$.paths.*[get,put,post,patch,delete,options,head].parameters.[?(@.name == 'api-version')]",
```